src/llm/eventToTags.ts

```diff
@@ -107,26 +107,15 @@ const CONTENT_TAG_PROMPT =
   Your answer must begin with: "Out of the the tags [${ACCEPTABLE_CONTENT_TAGS.join(", ")}]..."
 `;
 
-const EVENT_CONTENT_TAG_FUNCTION: ChatCompletionFunctions = {
-  name: "tag_event_content",
-  description: "Add content tag to event",
-  parameters: {
-    type: "object",
-    properties: {
-      content_tag_1: {
-        type: "string",
-        description: "The first tag of the content of the event. (not necessary)",
-        enum: ACCEPTABLE_CONTENT_TAGS
-      },
-      content_tag_2: {
-        type: "string",
-        description: "The second tag of the content of the event (not necessary).",
-        enum: ACCEPTABLE_CONTENT_TAGS
-      }
-    },
-    require: []
-  }
-};
+const EVENT_CONTENT_TAG_GRAMMAR = dedent`
+  content-tag-1 ::= "\"EECS\"" | "\"AI\"" | "\"Math\"" | "\"Biology\"" | "\"Finance\"" | "\"Career\"" | "\"East Asian\"" | "\"Religion\"" | "\"Queer\""
+  content-tag-1-kv ::= "\"content_tag_1\"" space ":" space content-tag-1
+  content-tag-1-rest ::= ( "," space content-tag-2-kv )?
+  content-tag-2 ::= "\"EECS\"" | "\"AI\"" | "\"Math\"" | "\"Biology\"" | "\"Finance\"" | "\"Career\"" | "\"East Asian\"" | "\"Religion\"" | "\"Queer\""
+  content-tag-2-kv ::= "\"content_tag_2\"" space ":" space content-tag-2
+  root ::= "{" space  (content-tag-1-kv content-tag-1-rest | content-tag-2-kv )? "}" space
+  space ::= " "?
+`
```

Comment:
I noticed you went with a grammar that expects an object of the form
```
{
"content-tag-1": value,
"content-tag-2": value
}
```
Is content-tag-2 optional according to the grammar? During testing it seemed to correctly identify one tag and unnecessarily have a second. I think the object format over the list may be better, but only if "content-tag-2" can be optional.